docs/modules/vearch.md

```diff
@@ -0,0 +1,57 @@
+# Vearch
+
+Since testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go/releases/tag/v0.29.0"><span class="tc-version">:material-tag: v0.29.0</span></a>
```

Comment:
I think you copy pasted from another module. I'd suggest running the module generator tool and check what files it produces:
```suggestion
Not available until the next release of testcontainers-go <a href="https://github.com/testcontainers/testcontainers-go"><span class="tc-version">:material-tag: main</span></a>
```